recipes/kartothek/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "kartothek" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 59b872febe0acee7fbead8b07f012a0ab631d760aa8f03e8650d2841765fac19
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - setuptools_scm
+    - python
+  run:
+    - dask
+    - decorator
+    - funcsigs
+    - msgpack-python >=0.5.2
+    - numpy !=1.15.0,!=1.16.0
+    - pandas >=0.23.0
+    - pyarrow >=0.11.1
+    - python
```

Comment:
I guess we should set `python>=3.6`

Reply:
or is this automagically discovered?

Reply:
Ah, correct!

Reply:
@fjetter fixed in the last commit. Also made this a `noarch: python` package as the binary content of the package doesn't vary by the python version during the build.